pkg/oc/cli/admin/mustgather/mustgather.go

```diff
@@ -108,7 +108,8 @@ func (o *MustGatherOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, arg
 		}
 		var ok bool
 		if o.Image, ok = util.ResolveLatestTaggedImage(imageStream, "latest"); !ok {
-			return fmt.Errorf("unable to to resolve must-gather image")
+			o.Image = "quay.io/openshift/origin-must-gather:latest"
```

Comment:
your idea of trying a rolling tag is a good one.

Maybe you can try to parse out from clusterversion